### PR TITLE
Create garden

### PR DESCRIPTION
**Pull request by djjxx1**
## Information

- [ ] This PR fixes a formatting issue.
- [ ] This PR corrects the information in an existing guide.
- [x] This PR adds a new guide.

## Checklist

- [ ] I have searched the open pull requests for duplicates.
- [ ] I am sure the guide is formatted properly.

## Licensing

- [x] I agree that my contributions shall be licensed under the [Creative Commons Attribution-ShareAlike 4.0 International License](https://creativecommons.org/licenses/by-sa/4.0/),
  and that SkyKings may distribute my contributions under the same license on their website.

### CODE REVIEW

**Comment by plun1331** [comment]
@djjxx1 Please check the "Licensing" box in your PR description

**Review by M4aXim — CHANGES_REQUESTED** [review]
Please correct alwways to always

**Review by DaytonWallace — DISMISSED** [review]

File: guides/garden.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+tags:
+  - Farming
+  - Garden
+  - Money Making Method
+```
+
+Garden is a great money making method and also a great way to earn skyblock xp. This guide talks about how to level up 
+your garden level, what to do with visitors and good money making methods
+
+Part 1: Leveling up your Garden
```
Comment (plun1331) [inline guides/garden.md]:
```suggestion
## Part 1: Leveling up your Garden
```

File: guides/garden.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+To level up your garden you need to one, unlock crop milestones, or two, complete visitor offers. At the start, you should
+get your crop milestones to at least 16 in all of them. I suggest you build a mega farm so you can farm as many crops at a
+time. 
+
+Part 1.5: Visitors
```
Comment (plun1331) [inline guides/garden.md]:
```suggestion
### Visitors
```

File: guides/garden.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  - Garden
+  - Money Making Method
+```
+
+Garden is a great money making method and also a great way to earn skyblock xp. This guide talks about how to level up 
+your garden level, what to do with visitors and good money making methods
+
+Part 1: Leveling up your Garden
+
+To level up your garden you need to one, unlock crop milestones, or two, complete visitor offers. At the start, you should
```
Comment (plun1331) [inline guides/garden.md]:
```suggestion
To level up your garden you need to either unlock crop milestones or complete visitor offers. At the start, you should
```

File: guides/garden.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+get your crop milestones to at least 16 in all of them. I suggest you build a mega farm so you can farm as many crops at a
+time. 
+
+Part 1.5: Visitors
+
+With visitors, you should accept all of the unique visitors, thevisitors with cheep offers, legendarys (they are
```
Comment (plun1331) [inline guides/garden.md]:
```suggestion
With visitors, you should accept all of the unique visitors, visitors with cheap offers, legendaries (they are 
```

File: guides/garden.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+Part 1: Leveling up your Garden
+
+To level up your garden you need to one, unlock crop milestones, or two, complete visitor offers. At the start, you should
+get your crop milestones to at least 16 in all of them. I suggest you build a mega farm so you can farm as many crops at a
+time. 
+
+Part 1.5: Visitors
+
+With visitors, you should accept all of the unique visitors, thevisitors with cheep offers, legendarys (they are 
+profit almost all of the time) and visitors that have crops you can farm.Getting to garden 10 is pretty easy but every
```
Comment (plun1331) [inline guides/garden.md]:
```suggestion
profit almost all of the time) and visitors that have crops you can farm. Getting to garden 10 is pretty easy but every 
```

File: guides/garden.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+get your crop milestones to at least 16 in all of them. I suggest you build a mega farm so you can farm as many crops at a
+time. 
+
+Part 1.5: Visitors
+
+With visitors, you should accept all of the unique visitors, thevisitors with cheep offers, legendarys (they are 
+profit almost all of the time) and visitors that have crops you can farm.Getting to garden 10 is pretty easy but every 
+level after 10 needs 10k garden xp to level up.
+
+Part 2: Money Making Methods
```
Comment (plun1331) [inline guides/garden.md]:
```suggestion
## Part 2: Money Making Methods
```

File: guides/garden.md [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+
+There are many good money making methods and some of them do require luck. The first money making method is farming in
+general. The best crop to farm is nether wart with a mooshroom cow as you get a lot from the nether wart and it has a 
+decent price. Other crops still make a lot of money but nether wart is the best The second money making method is
```
Comment (plun1331) [inline guides/garden.md]:
```suggestion
decent price. Other crops still make a lot of money but nether wart is the best.

The second money making method is
```

File: guides/garden.md [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+getting lucky with visitors. Visitors have a very small chance to drop an overgrown grass and a green bandana, each
+going for decently high prices. The third and final money making method of this guide is farming mushrooms for
```
Comment (plun1331) [inline guides/garden.md]:
```suggestion
going for decently high prices. 

The third and final money making method of this guide is farming mushrooms for
```

File: guides/garden.md [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+burrowing spores. Burrowing spores are slowly going down in price, but with the mushrooms you farm and a little
+bit of luck you can get lucky with it. Farming mushrooms is also a great way to gain farming xp.
+
+Part 3: The Composter
```
Comment (plun1331) [inline guides/garden.md]:
```suggestion
## The Composter
```

**Review by plun1331 — CHANGES_REQUESTED** [review]

**Review by plun1331 — APPROVED** [review]

**Review by DaytonWallace — APPROVED** [review]

**Pull request merged**